docs/releases.md

```diff
@@ -1,6 +1,54 @@
 # Ixia-c Release Notes and Version Compatibility
 
-## Release  v0.0.1-2678 (Latest)
+## Release  v0.0.1-2727 (Latest)
+> 24th March, 2022
+
+#### About
+
+This build implements received(rx) rate statistics of port metrics. It also has auto destination mac learning support using choice as `auto` in ethernet destination mac of packet header.
+
+#### Build Details
+
+| Component                     | Version       |
+|-------------------------------|---------------|
+| Open Traffic Generator API    | [0.7.8](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.7.8/openapi.yaml)         |
+| snappi                        | [0.7.18](https://pypi.org/project/snappi/0.7.18)        |
+| gosnappi                      | [0.7.18](https://pkg.go.dev/github.com/open-traffic-generator/snappi/gosnappi@v0.7.18)        |
+| ixia-c-controller             | [0.0.1-2727](https://hub.docker.com/r/ixiacom/ixia-c-controller/tags)    |
```

Comment:
update latest controller build when available

Reply:
sure will do that